libs/pageserver_api/src/models.rs

```diff
@@ -165,8 +165,6 @@ pub enum TimelineState {
     /// The timeline was previously Loading or Active but is shutting down.
     /// It cannot transition back into any other state.
     Stopping,
-    /// The timeline is broken and not operational (previous states: Loading or Active).
-    Broken,
 }
```

Comment:
note that in #4384 this is being used and expanded

Reply:
To elaborate, it is used when background deletion fails.